From 923c4ca79daa18e1cab444be7ee52b4bfcfda309 Mon Sep 17 00:00:00 2001
From: Guilhem Saurel <guilhem.saurel@laas.fr>
Date: Fri, 30 Sep 2022 17:04:12 +0200
Subject: [PATCH] CMake: force python 3 for noetic

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 361b1b3..cad3d58 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -36,6 +36,7 @@ if(BUILD_TESTING)
 endif(BUILD_TESTING)
 
 if(BUILD_PYTHON_INTERFACE)
+  set(PYTHON_EXECUTABLE "/usr/bin/python")
   add_project_dependency(dynamic-graph-python 4.0.6 REQUIRED)
 endif(BUILD_PYTHON_INTERFACE)
 
-- 
2.25.1

